shiny/express/ui/__init__.py

```diff
@@ -0,0 +1,300 @@
+    "Tag",
+    "TagChild",
+    "TagAttrs",
+    "TagAttrValue",
+    "tags",
+    "HTML",
+    "head_content",
+    "a",
+    "br",
+    "code",
+    "div",
+    "em",
+    "h1",
+    "h2",
+    "h3",
+    "h4",
+    "h5",
+    "h6",
+    "hr",
+    "img",
+    "p",
+    "pre",
+    "span",
+    "strong",
+    "tags",
+    # Imports from ...ui
+    "AccordionPanel",
+    "AnimationOptions",
+    "CardItem",
+    "ShowcaseLayout",
+    "Sidebar",
+    "SliderStepArg",
+    "SliderValueArg",
+    "ValueBoxTheme",
+    "card_header",
+    "card_footer",
+    "download_button",
+    "download_link",
+    "brush_opts",
+    "click_opts",
+    "dblclick_opts",
+    "help_text",
+    "hover_opts",
+    "include_css",
+    "include_js",
+    "input_action_button",
+    "input_action_link",
+    "input_checkbox",
+    "input_checkbox_group",
+    "input_switch",
+    "input_radio_buttons",
+    "input_date",
+    "input_date_range",
+    "input_file",
+    "input_numeric",
+    "input_password",
+    "input_select",
+    "input_selectize",
+    "input_slider",
+    "input_text",
+    "input_text_area",
+    "insert_accordion_panel",
+    "remove_accordion_panel",
+    "update_accordion",
+    "update_accordion_panel",
+    "update_sidebar",
+    "update_action_button",
+    "update_action_link",
+    "update_checkbox",
+    "update_switch",
+    "update_checkbox_group",
+    "update_radio_buttons",
+    "update_date",
+    "update_date_range",
+    "update_numeric",
+    "update_select",
+    "update_selectize",
+    "update_slider",
+    "update_text",
+    "update_text_area",
+    "update_navs",
+    "update_tooltip",
+    "update_popover",
+    "insert_ui",
+    "remove_ui",
+    "markdown",
+    "modal_button",
+    "modal",
+    "modal_show",
+    "modal_remove",
+    "notification_show",
+    "notification_remove",
+    "output_plot",
+    "output_image",
+    "output_text",
+    "output_text_verbatim",
+    "output_table",
+    "output_ui",
+    "Progress",
+    "output_data_frame",
+    "value_box_theme",
+    # Imports from ._cm_components
+    "set_page",
+    "sidebar",
+    "layout_sidebar",
+    "layout_column_wrap",
+    "column",
+    "row",
+    "card",
+    "accordion",
+    "accordion_panel",
+    "navset",
+    "navset_card",
+    "nav",
+    "value_box",
+    "panel_well",
+    "panel_conditional",
+    "panel_fixed",
+    "panel_absolute",
+    "page_fluid",
+    "page_fixed",
+    "page_fillable",
+    "page_sidebar",
+    "page_navbar",
+)
+
+
+_known_missing = {
+    # Items from shiny.ui that don't have a counterpart in shiny.express.ui
+    "shiny.ui": (
+        "nav_control",
+        "nav_menu",
+        "nav_spacer",
+        "navset_bar",
+        "navset_card_pill",
+        "navset_card_tab",
+        "navset_card_underline",
+        "navset_hidden",
+        "navset_pill",
+        "navset_pill_card",
+        "navset_pill_list",
+        "navset_tab",
+        "navset_tab_card",
+        "navset_underline",
```

Comment:
Are we not including these because express has `navset()` and `navset_card()` now? 

For context, when I added `navset()`/`navset_card()`, the primary motivation was coming trying to keep `shiny.express.ui`'s API minimal (I still think that could be a valuable thing, but would come at the cost of needing two UI imports).

Anyway, assuming we go in this direction (of `shiny.express.ui` basically having everything that `shiny.ui` has), I think I'd be favor of removing `navset()`/`navset_card()` and exposing all these as context managers.

Reply:
I don't have strong opinions about this either way. I'm fine with removing `navset()` and `navset_card()` and adding the others as context managers.

Reply:
I replaced `navset` and `navset_card` with the specific components in 364cbad6.